.travis.yml

```diff
@@ -1,20 +1,44 @@
 language: ruby
 rvm:
   - 2.3.1
+
 branches:
   only:
     - master
     - develop
-script: bundle exec rspec --tag ~broken_in_ci
+
+services:
+  - mysql
+  - postgresql
+
+env:
+  - DB=sqlite
+  - DB=mysql
+  - DB=postgres
+
+before_install:
+  - if [[ $DB = 'sqlite' ]]; then echo "gem 'sqlite3'" > Gemfile.local; fi
+  - if [[ $DB = 'mysql' ]]; then echo "gem 'mysql2'" > Gemfile.local; fi
+  - if [[ $DB = 'postgres' ]]; then echo "gem 'pg'" > Gemfile.local; fi
+
 install:
   - bundle install --jobs=3 --retry=3
-  - bundle exec sqlite3 --version
+
+before_script:
   - bundle exec rake config
+  - cp config/database.yml.travis-$DB config/database.yml
+  - if [[ $DB = 'sqlite' ]]; then bundle exec rake db:create; fi
+  - if [[ $DB = 'mysql' ]]; then mysql -e 'CREATE DATABASE dpn;'; fi
+  - if [[ $DB = 'postgres' ]]; then psql -c 'CREATE DATABASE dpn;' -U postgres; fi
```

Comment:
Wondering if this `psql` needs to set the owner of the db?  On my laptop, I had to use something like
```
createdb -T template0 --owner=dpnAdmin dpn_development
```
Maybe that's only required when the user is not `postgres`.